content/posts/2025-07-17-whats-new-1.9/index.md

```diff
@@ -0,0 +1,197 @@
+
+```yaml
+spec:
+  defaultNetwork:
+    ovnKubernetesConfig:
+      gatewayConfig:
+        routingViaHost: true  # change
+      ipsecConfig:
+        mode: Full  # change
+      mtu: 8855  # update with your MTU value and below
+  migration:  # add this section
+    mtu:
+      network:
+        from: 8901
+        to: 8855
+      machine:
+        to: 9001
+```
+
+Make the two changes with the comment "change" to enable IPsec.  Reduce the network MTU by 46 bytes, which are needed by the ESP header to do IPsec encryption.  My current network MTU value was 8901 (using jumbo frames) and was reduced to 8855.  You also need to provide the machine or physical MTU on the interface even though it won't be changed.
+
+To get the cluster or network MTU, enter `oc get networks.operator.openshift.io -o yaml | grep mtu`.  To get the machine MTU, look at the example below and issue the commands, but replace with your pod name.
+
+```
+$ oc project openshift-ovn-kubernetes
+$ oc get pods
+NAME                                     READY   STATUS    RESTARTS        AGE
+ovnkube-control-plane-595bf6d946-9gjjb   2/2     Running   2 (4h48m ago)   5h3m
+ovnkube-node-5h682                       8/8     Running   1 (4h58m ago)   4h58m
+ovnkube-node-94z84                       8/8     Running   0               5h3m
+
+$ oc rsh ovnkube-node-5h682
+
+sh-5.1# ip link show
+...
+2: ens5: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 9001 qdisc mq master ovs-system state UP mode DEFAULT group default qlen 1000
+    link/ether 06:a5:5a:cd:ff:6b brd ff:ff:ff:ff:ff:ff
+    altname enp0s5
+```
+
+Look for the interface such as *ensX* and the *mtu* value.  Here, it's 9001.
+
+Now you can start the migration and IPsec enablement.  During this process, you will lose connection to the cluster.  It may be five minutes or longer before it automatically connects back.  When it does, IPsec pods should be running.
+
+```
+$ oc get pods
+NAME                                     READY   STATUS    RESTARTS      AGE
+ovn-ipsec-containerized-jpnxr            1/1     Running   1             4m14s
+ovn-ipsec-containerized-w6r9c            1/1     Running   1             4m14s
+ovn-ipsec-host-27j49                     2/2     Running   2             4m14s
+ovn-ipsec-host-jjb54                     2/2     Running   2             4m14s
+ovnkube-control-plane-595bf6d946-9gjjb   2/2     Running   6             5h24m
+ovnkube-node-dfhck                       7/8     Running   7 (16s ago)   3m32s
+ovnkube-node-dtq28                       7/8     Running   8             4m11s
+```
+
+However, Network Observability will report an "AxiosError" if using Loki. Wait at least another 10 minutes or more until the service CA (Certificate Authority) is updated.
+
+
+## Flowlogs-pipeline filter
```

Comment:
you asked above if there's a doc about the fields, it's actually the flows format reference: https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/network_observability/json-flows-format-reference